templates/vue/tests/unit/Notification.spec.js

```diff
@@ -0,0 +1,26 @@
+import { shallowMount } from '@vue/test-utils'
+import Notification from '@/components/Notification.vue'
```

Comment:
what is this `from '@/...'` notation?

Reply:
it means: "*from the root of the project*", but in this case it's not needed. I'll remove it.

Reply:
Whoa cool! Thanks for teaching me this!